### ISSUE TITLE
计算面的夹角的时候，为什么要给法向量加一个-1？

### ISSUE THREAD

**Issue by zzt93** [k2cu8/pyOMT#4]
如下demo2的代码：

    '''compute angles'''
    P = p_s.h_P      
    nm = torch.cat([P, -torch.ones(p_s.num_P,1)], dim=1)
    nm /= torch.norm(nm,dim=1).view(-1,1)
    cs = torch.sum(nm[I_all[0,:],:] * nm[I_all[1,:],:], 1) #element-wise multiplication
    cs = torch.min(torch.ones([cs.shape[0]]), cs)
    theta = torch.acos(cs)

这里为什么要cat一个-1？不需要-1就可以直接计算了呀？
    nm = torch.cat([P, -torch.ones(p_s.num_P,1)], dim=1)
